Update Vocalink data sets to v640 valid as of 12/10/2020

solanki-das opened the pull request.

franciscocardoso reviewed (changes requested):
Hi, 

Can you please follow this [example](https://github.com/uphold/uk-modulus-checking/pull/16/files)? In order to correctly update this module you need to do 3 things:
1. Update the data file, which you have already done;
2. Update the loadValacdos function;
3. Update the docs folder with the correspondent version;

solanki-das commented:
Hi,
As suggested, I have 
1. Updated the loadValacdos function;
2. Updated the docs folder with the correspondent version;

solanki-das commented:
Hi,
As suggested, I have
1. Updated the loadValacdos function;
2. Updated the docs folder with the correspondent version;
Could you please check.

Thanks
Solanki

From: Francisco Cardoso <notifications@github.com>
Date: Tuesday, 24 November 2020 at 18:58
To: uphold/uk-modulus-checking <uk-modulus-checking@noreply.github.com>
Cc: Solanki Das <solanki.das1@engineering.digital.dwp.gov.uk>, Author <author@noreply.github.com>
Subject: Re: [uphold/uk-modulus-checking] Update Vocalink data sets to v640 valid as of 12/10/2020 (#23)

@franciscocardoso requested changes on this pull request.

Hi,

Can you please follow this example<https://github.com/uphold/uk-modulus-checking/pull/16/files>? In order to correctly update this module you need to do 3 things:

  1.  Update the data file, which you have already done;
  2.  Update the loadValacdos function;
  3.  Update the docs folder with the correspondent version;

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/uphold/uk-modulus-checking/pull/23#pullrequestreview-537816423>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ARZXDKPKZZ5DAGDL2RELRULSRP65DANCNFSM4T7KOXLA>.

franciscocardoso commented:
You are removing the pdf file. Please replace it with the [new one](https://www.vocalink.com/media/4749/vocalink-validating-account-numbers-v640.pdf) instead.
Also, please don't forget to squash the commits into one. If you are not aware how to do it you can read [this](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).

solanki-das commented:
Hi,

I have replaced the new file with the old version and squash the commits into one. Please find the attached screenshot of the log.

Thanks
Solanki

From: Francisco Cardoso <notifications@github.com>
Date: Thursday, 26 November 2020 at 17:26
To: uphold/uk-modulus-checking <uk-modulus-checking@noreply.github.com>
Cc: Solanki Das <solanki.das1@engineering.digital.dwp.gov.uk>, Author <author@noreply.github.com>
Subject: Re: [uphold/uk-modulus-checking] Update Vocalink data sets to v640 valid as of 12/10/2020 (#23)

You are removing the pdf file. Please replace it with the new one<https://www.vocalink.com/media/4749/vocalink-validating-account-numbers-v640.pdf> instead.
Also, please don't forget to squash the commits into one. If you are not aware how to do it you can read this<https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/uphold/uk-modulus-checking/pull/23#issuecomment-734416074>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ARZXDKLF7ONV62EPI4FJZ5DSR2FVPANCNFSM4T7KOXLA>.

solanki-das commented:
Hi,

I have already made the changes requested by you. Could you please review and let me know whether any other changes are required.

Regards
Solanki.

From: Solanki Das <solanki.das1@engineering.digital.dwp.gov.uk>
Date: Monday, 30 November 2020 at 15:53
To: uphold/uk-modulus-checking <reply@reply.github.com>
Subject: Re: [uphold/uk-modulus-checking] Update Vocalink data sets to v640 valid as of 12/10/2020 (#23)
Hi,

I have replaced the new file with the old version and squash the commits into one. Please find the attached screenshot of the log.

Thanks
Solanki

From: Francisco Cardoso <notifications@github.com>
Date: Thursday, 26 November 2020 at 17:26
To: uphold/uk-modulus-checking <uk-modulus-checking@noreply.github.com>
Cc: Solanki Das <solanki.das1@engineering.digital.dwp.gov.uk>, Author <author@noreply.github.com>
Subject: Re: [uphold/uk-modulus-checking] Update Vocalink data sets to v640 valid as of 12/10/2020 (#23)

You are removing the pdf file. Please replace it with the new one<https://www.vocalink.com/media/4749/vocalink-validating-account-numbers-v640.pdf> instead.
Also, please don't forget to squash the commits into one. If you are not aware how to do it you can read this<https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/uphold/uk-modulus-checking/pull/23#issuecomment-734416074>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ARZXDKLF7ONV62EPI4FJZ5DSR2FVPANCNFSM4T7KOXLA>.

franciscocardoso reviewed (approved):
Hi @solanki-das 
I took the liberty of squashing the commits.
Thanks for your contribution and effort to keep this module up to date.
